movie-smt-ui/src/components/MovieCard/index.tsx

```diff
@@ -0,0 +1,75 @@
+import { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+import styles from "./MovieCard.module.scss";
+import { Movie } from "../../utils/types";
+import {
+  Card,
+  CardActionArea,
+  CardContent,
+  CardMedia,
+  createTheme,
+  Skeleton,
+  Typography,
+} from "@mui/material";
+import { ThemeProvider } from "@emotion/react";
+
+const theme = createTheme({
+  palette: {
+    background: {
+      paper: "#242424",
+    },
+    text: {
+      primary: "#CFD3FD",
+      secondary: "#46505A",
+    },
+    action: {
+      active: "#001E3C",
+    },
+  },
+});
+
```

Comment:
This kind of logic is very counter productive, we should have a central theme for the whole website where we can then pull different colors from and then use these variables everywhere, this way not only does the look stay consistent but if we wanna change it it gets changed everywhere. In general avoid custom css for each component as much as possible

Reply:
https://mui.com/material-ui/customization/theming/